### ISSUE TITLE
Autocomplete from Chrome are blocking calendar input display

### ISSUE THREAD

**Issue by danielinki** [primefaces/primeng#7809]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report => Search github for a similar issue or PR before submitting
[ ] feature request => Please check if request is not on the roadmap already https://github.com/primefaces/primeng/wiki/Roadmap
[ ] support request => Please do not submit support request here, instead see http://forum.primefaces.org/viewforum.php?f=35
```

**Current behavior**
Autocomplete from Chrome are blocking calendar input display
<img width="288" alt="Screenshot 2019-06-14 at 11 02 54 AM" src="https://user-images.githubusercontent.com/19463277/59482670-0c0d1a00-8e94-11e9-98e9-7996063757db.png">

**Expected behavior**
hide the autocomplete

**Minimal reproduction of the problem with instructions**
Open p-calendar with chrome browser

**What is the motivation / use case for changing the behavior?**
the autocomplete suggestions overlap the calendar. looks like chrome is ignoring autocomplete="off" now

**Please tell us about your environment:**

* **Angular version:** 5.2.11

* **PrimeNG version:** 5.2.6

* **Browser:** Version 75.0.3770.90 (Official Build) (64-bit)

 
* **Language:** [all | TypeScript X.X | ES6/7 | ES5]

* **Node (for AoT issues):** `node --version` =   10.16.0

**Comment by mertsincan** [comment]
Hi @danielinki,

I couldn't replicate it. Which device are you using? chrome in Mobile or PC. Also, could you please try the following code?

```
<form autocomplete="off">
```

**Comment by danielinki** [comment]
@mertsincan you can try to enable this on chrome setting. it will load the cache for the field, maybe need to try several times to get it store in cache
<img width="1070" alt="Screenshot 2019-06-19 at 5 08 03 PM" src="https://user-images.githubusercontent.com/19463277/59822757-16c52480-9356-11e9-82f1-b56551558494.png">

**Comment by cagataycivici** [comment]
Unable to replicate with PrimeNG 9, if the issue persists please create a new ticket with a test case reproducing the issue e.g. stackblitz or a github repo and it will be reviewed by our team once again.

The calendar input has autocomplete="off" in the current code.

**Comment by Gighub** [comment]
May I ask if this was ever resolved as I am experiencing the same situation?

When I click the calendar icon/button the date picker model appears however, I am presented with a list of previously entered dates by Chrome which sit on top of the date picker model.

I have tried to use autocomplete="false" and autocomplete="off" but this has not prevented the Chrome autocomplete from showing.

I have attached a screenshot.

My set up is, Macmini, Chrome - Version 107.0.5304.110 (Official Build) (arm64)

![Screenshot 2022-11-25 at 11 50 11](https://user-images.githubusercontent.com/14055858/203967658-4e53dc09-17d6-4155-88a8-2067b4350bdc.png)

Thank you in advance for your help.
